styles.css

```diff
@@ -107,3 +107,450 @@ a {
   width: 1.8rem;
   height: 1.87rem;
 }
+
+.recent-work {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 4rem 1.5rem;
+  gap: 3.6rem;
+  margin-bottom: 0;
+}
+
+.recent-work-projects {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 1.5rem;
+}
+
+.recent-work-projects-text {
+  font-weight: 700;
+  font-size: 2.5rem;
+  line-height: 3.2rem;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #172b4d;
+}
+
+.indicator {
+  width: 48px;
+  height: 4px;
+  background: #36b37e;
+  border-radius: 1.5rem;
+}
+
+.card {
+  display: grid;
+  flex-direction: column;
+  align-items: center;
+  padding: 0.2rem 0.1rem;
+  gap: 1.5rem;
+}
+
+.card-enable {
+  width: 327px;
+  height: 474px;
+  display: grid;
+}
+
+.img-placeholder {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  justify-content: flex-end;
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.project-text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1rem;
+  gap: 1rem;
+  background: #fff;
+  border-radius: 0.5rem;
+}
+
+.project-text ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.project-text li {
+  margin: 0;
+  padding: 0;
+}
+
+.project-title {
+  font-weight: 600;
+  font-size: 1.5rem;
+  line-height: 2em;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #3a4a42;
+  margin: 0;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.tag {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0.5rem 0.7rem;
+  background: #ebf0ee;
+  border-radius: 0.2rem;
+}
+
+.tag-text {
+  font-weight: 600;
+  font-size: 0.7rem;
+  line-height: 1em;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+  margin: 0.5rem;
+}
+
+.see-project {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding-top: 1.8rem;
+}
```

Comment:
@- [ ] Kindly make sure your project cards stick to the ones from the Figma design to match the project's requirements. You can see that your design is different from the Figma template, especially the `height` of `background color block` and `text`, spacing between card content, the padding on the `See Project` button, `font size` on technologies text on each category(e.g: Ruby on rails, CSS, Rspec,...),...

| Figma Design | Figma Design |
| ------------------- | --------------------- | 
| ![project_card_f3](https://user-images.githubusercontent.com/83042742/185718044-fbd1638e-d756-42d1-87d6-d3203925087c.png) | ![Screenshot from 2022-08-20 01-12-43](https://user-images.githubusercontent.com/83042742/185718231-045f550b-a172-420c-b78d-6bb4ea9dc72a.png) |